RiotSharp/StaticDataEndpoint/Champion/Enums/ParTypeStatic.cs

```diff
@@ -72,6 +72,14 @@ public enum ParTypeStatic
         /// <summary>
         /// Wind (Yasuo).
         /// </summary>
-        Wind
+        Wind,
+
+        Courage,
+
+        CrimsonRush,
+
+        Flow,
+
+        Fury,
```

Comment:
Could you add the corresponding comments (concerned champions would be great) just like it's been done for the others?